algorithms/pact/pact_ops.py

```diff
@@ -113,7 +113,7 @@ def forward(ctx, x, mul, add, div, signed, n_levels_out, cmsis_requant):
             else:
                 y = x * mul + add
                 # Avoid round to even behaviour, friggin pytorch
-                y = torch.floor((y / div) + 0.5)
+                y = torch.floor((y / div))
```

Comment:
I think there is a separate PR for this, I don't mind whether we integrate this one or the other.
Is this the correct arithmetic for Dory?